sdk/search/azure-search/pom.xml

```diff
@@ -28,13 +28,14 @@
   <dependencies>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-core</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
     </dependency>
     <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-core-http-netty</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.10.1</version> <!-- {x-version-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
+      <scope>compile</scope>
```

Comment:
This shouldn't be added.

Reply:
The tests complains about missing module.

Reply:
Double check the modules dependencies. Remove all unnecessary ones.

Reply:
Removed. 

The jackson core module is transitive from azure-core instead of azure-core-netty. Just added azure-core dependency back to make tests work